src/mono/wasm/runtime/types/internal.ts

```diff
@@ -439,8 +439,8 @@ export declare interface EmscriptenModuleInternal {
     ENVIRONMENT_IS_PTHREAD?: boolean;
     wasmModule: WebAssembly.Instance | null;
     ready: Promise<unknown>;
-    asm: { memory?: WebAssembly.Memory };
-    wasmMemory?: WebAssembly.Memory;
+    asm: any;
+    getMemory(): any;
```

Comment:
is the return type `WebAssembly.Memory` ?

Reply:
For asm["memory"] in MT, no. Will find out for getMemory.